capplets/accessibility/at-properties/dm-util.c

```diff
@@ -0,0 +1,162 @@
+/* vi: set sw=4 ts=4 wrap ai: */
+/*
+ * dm-util.h: This file is part of ____
+ *
+ * Copyright (C) 2019 yetist <yetist@yetipc>
+ *
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ * */
+
+#include <gio/gio.h>
+#include "dm-util.h"
+
+static GDBusProxy *get_sys_proxy (void)
+{
+    GError            *error = NULL;
+    GDBusProxy *proxy = NULL;
+
+    if (proxy == NULL) {
```

Comment:
this test seems useless as `proxy` will always be `NULL` here as the variable is not static and initialized just above as `NULL`